src/displayapp/screens/settings/SettingTimeFormat.cpp

```diff
@@ -74,6 +76,9 @@ void SettingTimeFormat::UpdateSelected(lv_obj_t* object, lv_event_t event) {
         if (i == 1) {
           settingsController.SetClockType(Controllers::Settings::ClockType::H24);
         };
+        if (i == 2) {
+          settingsController.SetClockType(Controllers::Settings::ClockType::UTC);
+        };
```

Comment:
This would probably look nicer as an `else if` or `switch`?

Reply:
Agreed.